plugins/plugin-client-common/src/components/Views/Terminal/ScrollableTerminal.tsx

```diff
@@ -537,6 +545,8 @@ export default class ScrollableTerminal extends React.PureComponent<Props, State
     this.setState(curState => {
       const idx = this.findSplit(this.state, sbuuid)
       if (idx >= 0) {
+        curState.splits[idx].blocks.forEach(this.removeWatchableBlock)
```

Comment:
doesn't this need either a `this.removeWatchableBlock.bind(this)` ? or `block => this.removeWatchableBlock(block)`?

Reply:
i guess not? is this a change to nodejs?